app/views/transactions/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+<%= form_with model: [@category, @category.transactions.create]  do |form| %>
+  <% if transaction.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(transaction.errors.count, "error") %> prohibited this transaction from being saved:</h2>
+
+      <ul>
+        <% transaction.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, class: "form-label", style: "display: block" %>
+    <%= form.text_field :name, class: "form-control" %>
+  </div>
+
+  <div>
+    <%= form.label :amount, class: "form-label", style: "display: block" %>
+    <%= form.number_field :amount, class: "form-control" %>
+  </div><br>
+
+  <div class="sign-buttons">
+    <%= form.submit class: "btn btn-success", id: "btn-log" %>
```

Comment:
- [ ] When I attempt to submit the form without filling out the fields, I got the error on the screenshot below instead of validation messages



![Screenshot from 2023-02-25 18-03-22](https://user-images.githubusercontent.com/63087491/221370041-43371eb8-3034-4aa4-9264-54630e9fab40.png)